src/promptflow/tests/sdk_cli_test/conftest.py

```diff
@@ -83,6 +87,21 @@ def setup_local_connection(local_client, azure_open_ai_connection):
     _connection_setup = True
 
 
+@pytest.fixture
+def setup_experiment_table(local_client, azure_open_ai_connection):
```

Comment:
why we need `local_client` here?

Reply:
Oh, I was wrong about that, it's not required here.